### PR TITLE
io: read input from console

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**
Done with the first Exercise from I/O topic.

### CODE REVIEW

File: ex1/src/ConsoleReader.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ * Created by clouway on 16.05.16.
+ */
+
+/**
+ * Implements Reading interface
+ */
+public class ConsoleReader implements Reading {
+    private BufferedReader reader;
+
+    public ConsoleReader(InputStreamReader inStream) {
```
Comment (mgenov) [inline ex1/src/ConsoleReader.java]: You always have to pass upper level interface/abstract class. InputStreamReader looks like concrete implementation.

File: ex1/src/Reading.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+/**
+ * Created by clouway on 16.05.16.
+ */
+public interface Reading {
```
Comment (mgenov) [inline ex1/src/Reading.java]: Not sure that you use this interface somewhere.

**Comment by AlexanderVladimirov9090** [comment]
Fixed code on ex1 from I/O topic.

File: ex1/src/ConsoleStream.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import java.io.*;
+
+/**
+ * Created by clouway on 16.05.16.
+ */
+
+public class ConsoleStream {
```
Comment (mgenov) [inline ex1/src/ConsoleStream.java]: If you are passing InputStream it's not a ConsoleStream, cause InputStream could be taken from many places. 

Console, Socket, Memory and so on.

Comment (mgenov) [inline ex1/src/ConsoleStream.java]: For example:

The following statement is valid: 
```java
new ConsoleStream(new ByteArrayInputStream("foobarfoobar".getBytes())) {
}
```

**Comment by AlexanderVladimirov9090** [comment]
Bug Fixed for ex1.

File: ex1/src/ConvertedStream.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+     * @return
+     */
+    public float readFloat() {
+        float f = Float.parseFloat(stringFromInputStream());
+
+        return f;
+    }
+
+
+    private String stringFromInputStream() {
```
Comment (mgenov) [inline ex1/src/ConvertedStream.java]: Reading of this method doesn't sounds good to me. Probably something like: 

```java
String line = getNextLine();
// or 
String nextInput = readNextLine(); 
```

I name it line, cause you are calling actually "readLine".

**Comment by AlexanderVladimirov9090** [comment]
Renamed method and removed redundancy that passed my eyes in code.

**Comment by AlexanderVladimirov9090** [comment]
Done.

**Comment by mgenov** [comment]
LGTM ! 

Please squash your commits before pushing to upstream.

**Pull request merged**